### ISSUE TITLE
WiFi refresh not working

### ISSUE THREAD

**Issue by luca-gambarotto** [powenn/AltServer-Linux-PyScript#16]
I'm facing issues using WiFi refresh (option -w in the script). I can install AltStore on my iphone 14 without problems, and the refresh using usb (-t option) is working fine. The WiFi deamon instead gives me this error, and the phone can't find the AltServer to refresh the apps. How can I solve?
```

Enter OPTION to continue : No argument supplied, if you want for help, please use -h or --help
Not supplying ipa, running in server mode!
Running python3 command to advertise AltServer: from ctypes import *; dll = CDLL('libdns_sd.so'); sdRef = c_int(); flags = 0; interfaceIndex = 0; name = None; regtype = br'_altserver._tcp'; domain = None; host = None; port = 55246; txtLen = 17; txtRecord = b'\x10\x73\x65\x72\x76\x65\x72\x49\x44\x3D\x31\x32\x33\x34\x35\x36\x37'; ret = dll.DNSServiceRegister(byref(sdRef), flags, interfaceIndex, name, regtype, domain, host, port, txtLen, txtRecord, None, None); print('DNSServiceRegister result: %d' % ret); from threading import Event; Event().wait(); 
Starting netmuxd
Starting mDNS discovery for _apple-mobdev2._tcp.local with mdns
Listening on /var/run/usbmuxd
*** WARNING *** The program 'python3.10' uses the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/blog/projects/avahi-compat.html>
**DNSServiceRegister result: -65537**

```

**Comment by powenn** [comment]
Did you read the closed issues about wifi refresh ?
If not , check them first

**Comment by luca-gambarotto** [comment]
Hi powenn, yes I checked the past issues but I didn't find anyone with the same issue (**DNSServiceRegister result: -65537**).

I also didn't mentioned it in my first post, but avahi-daemon.service and avahi-daemon.socket are bot running fine.

**Comment by powenn** [comment]
What's your arch and linux distro ?

**Comment by luca-gambarotto** [comment]
I'm currently on Ubuntu 22.04. What do you mean by "arch distro"?

**Comment by powenn** [comment]
Architecture

**Comment by luca-gambarotto** [comment]
Oh, I see it. Ubuntu 22.04 x86_64

**Comment by powenn** [comment]
Can you install ipa with WiFI ?

**Comment by luca-gambarotto** [comment]
Just tested and unfortunately no, it says "0xe8008016 (The executable was signed with invalid entitlements.)". The .ipa file is working for sure, I installed it multiple times on Windows AltStore

**Comment by powenn** [comment]
Any error log in AltStore ?

**Comment by joshbober** [comment]
@luca-gambarotto just wondering, did you try running this in privileged (sudo) mode? I got a similar error (DNSServiceRegister result: -65533) while running this in a docker container. I was able to run it just fine with sudo in the cli (outside container), so I tried rebuilding the container with "Privileged mode" enabled and it started working. Hope this helps, I have spent like 20 hours this weekend trying to resolve this.
